BB-CSC/BB_Cloud_container_and_OS.md

```diff
@@ -0,0 +1,98 @@
+
+# Cloud contaier and OS
+
+## BB Tags(s)
+<!-- Tag(s) define in which area(s) (cloud, in-vehicle) the BB is executed, and what type of BB it is (tool, process, microservice) -->
+BB-CSC
+
+## Functional Clusters
+<!-- In which Functional Cluster the BB be located; if none of the existing fit new required -->
+Not specified
+
+## Layer
+<!-- AppLayer, MWLayer, OSLayer, HWLayer -->
+Not specified
+
+## Known Implementation
+
```

Comment:
- Blackberry QNX
- OxidOS
- Eclipse ThreadX
- RedHat In-Vehicle OS (announced)

Reply:
added